infra/main.bicep

```
@@ -30,13 +30,34 @@ param solutionUniqueText string = take(uniqueString(subscription().id, resourceG
   'westeurope'
   'uksouth'
 ])
-param location string = 'australiaeast'
+param location string
 
 // Restricting deployment to only supported Azure OpenAI regions validated with GPT-4o model
 @allowed(['australiaeast', 'eastus2', 'francecentral', 'japaneast', 'norwayeast', 'swedencentral', 'uksouth', 'westus'])
-@metadata({ azd: { type: 'location' } })
-@description('Optional. Location for all AI service resources. This should be one of the supported Azure AI Service locations.')
-param azureAiServiceLocation string = 'australiaeast'
+@metadata({
+  azd : {
+    type: 'location'
+    usageName : [
+      'OpenAI.GlobalStandard.gpt-4o, 150'
+    ]
+  }
+})
+@description('Required. Location for all AI service resources. This should be one of the supported Azure AI Service locations.')
+param aiDeploymentsLocation string
+
+@minLength(1)
+@description('Optional. Name of the GPT model to deploy:')
+param gptModelName string = 'gpt-4o'
+
+@description('Optional. Version of the GPT model to deploy. Defaults to 2024-08-06.')
+param gptModelVersion string = '2024-08-06'
+
+@minLength(1)
+@description('Optional. GPT model deployment type. Defaults to GlobalStandard.')
+param modelDeploymentType string = 'GlobalStandard'
```

For consistency, can we also call this parameter gptModelDelploymentType?
```suggestion
param gptModelDeploymentType string = 'GlobalStandard'
```

---

Updated it to gptModelDeploymentType